velox/connectors/hive/iceberg/tests/IcebergSplitReaderBenchmark.cpp

```diff
@@ -60,6 +60,20 @@ void IcebergSplitReaderBenchmark::writeToPositionDeleteFile(
   posDeletewriter.close();
 }
 
+std::vector<int64_t> IcebergSplitReaderBenchmark::makeRandomDeleteRows(
+    int32_t deleteRowsCount) {
+  std::mt19937 gen{0};
```

Comment:
This will generate same random number for every run, is this expected?

Reply:
@Yuhta Yes This is somewhat expected. But I'm ok if you think of it otherwise.